apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -876,7 +878,14 @@ get_account_transactions(Account, Req) ->
                                               Account),
             Res =
               lists:sort(
-                  fun({HeaderA, SignedTxA}, {HeaderB, SignedTxB}) ->
+                  fun({mempool, SignedTxA}, {mempool, SignedTxB}) ->
+                      TxA = aetx_sign:tx(SignedTxA),
+                      TxB = aetx_sign:tx(SignedTxB),
+                      {aetx:origin(TxA), aetx:nonce(TxA)} >
+                      {aetx:origin(TxB), aetx:nonce(TxB)};
```

Comment:
Do I understand correctly this line is meant to sort txs from same origin account in descending order of nonce? (It appears to me what it is doing based on code inspection.)

I understand this is not the first point in the function that introduced this approach, though I am trying to understand. From lists:sort/2 doc: "Fun(A, B) is to return true if A compares less than or equal to B in the ordering, otherwise false."

Other thing I am not sure about is equality. If order is meant to be in descending nonce order, I would expect `>=` not `>`.

(I picked the nonce though the principle may apply to all of the function.)

Reply:
> Do I understand correctly this line is meant to sort txs from same origin account in descending order of nonce? (It appears to me what it is doing based on code inspection.)
The transactions in this list are all with the same account somehow involved (as being one of aetx:accounts/1) but not mandatorary the origin (in the context of a spend_tx - the account could be the reciever as well).